Sweeper/View Controllers/Home/PinsListViewController.swift

```diff
@@ -39,10 +39,17 @@ class PinsListViewController: UIViewController, UITableViewDataSource, NVActivit
         super.viewWillAppear(animated)
         
         // refilter on every load in case pins were visited on the details page
+        let originalCount = pins.count
         pins = pins.filter { (pin: Pin) -> Bool in
             let visited = pin.visited ?? false
             return !visited
         }
+        
+        // reload table data if anything changed
+        if pins.count != originalCount {
```

Comment:
update originalCount or else it'll always reload

Reply:
Other than on L42?

Reply:
Oops misread that, looks good